dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -2431,6 +3149,13 @@
     "TextAnalysisAuthoringProjectSettings": {
       "description": "Represents the settings used to define the project behavior.",
       "type": "object",
+      "properties": {
+        "confidenceThreshold": {
```

Comment:
This property already exists as "confidenceThreshold" in the May GA APis for analyze-conversations, so I guess we need to keep the same name for consistency and to avoid confusion.